src/calculator.js

```diff
@@ -0,0 +1,84 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+const Button = (props) => {
+  const { label } = props;
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
+  </div>
+);
+
+Button.propTypes = {
+  label: PropTypes.string,
+};
```

Comment:
- [ ] Please create a directory called `components` and move your `calculator` component to the directory as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/205930084-4b07600a-dc0d-4abd-be8b-7dfd9faf0082.png)